include/alpaka/intrinsic/IntrinsicUniformCudaHipBuiltIn.hpp

```diff
@@ -79,6 +79,30 @@ namespace alpaka
 #endif
                 }
             };
+
+            //#############################################################################
+            template<>
+            struct Ffs<
+                IntrinsicUniformCudaHipBuiltIn>
+            {
+                //-----------------------------------------------------------------------------
+                __device__ static auto ffs(
+                    intrinsic::IntrinsicUniformCudaHipBuiltIn const & /*intrinsic*/,
+                    int value)
+                -> int
```

Comment:
why do you not return a `std::int32_t` here too? To have the same return type you have on the CPU?

Reply:
Yes, I should, thanks for pointing out.